grakn-graql/src/main/java/ai/grakn/graql/internal/gremlin/fragment/Fragment.java

```diff
@@ -99,16 +99,26 @@
     static final double COST_NODE_NOT_INTERNAL = -Math.log(1.1D);
     static final double COST_NODE_IS_ABSTRACT = -Math.log(1.1D);
 
+    public static final double SHARD_LOAD_FACTOR = 0.25 - 1.0; // there is at least one shard
```

Comment:
It's not totally clear what these numbers mean. What does 0.25 mean?

Reply:
By default I assume a shard is 25% full. So one shard mean 2500 instances, two shards = 12500, so on so forth